scripts/sync.py

```diff
@@ -139,14 +140,12 @@ def do(args, Lochness):
 
     # initialize (overwrite) metadata.csv using either REDCap or RPMS database
     if 'redcap' in args.input_sources or 'rpms' in args.input_sources:
-        if len(args.studies) == 1:
-            lochness.initialize_metadata(Lochness, args,
-                                         multiple_site_in_a_repo=False)
-        else:
-            # for ProNET and PRESCIENT, single REDCap and RPMS repo has
-            # information from multiple site
-            lochness.initialize_metadata(Lochness, args,
-                                         multiple_site_in_a_repo=True)
+        upenn_redcap = True if 'upenn' in args.input_sources else False
```

Comment:
Is `args.input_sources` a string?

Reply:
it's a list of strings, eg) `args.input_sources = ['xnat', 'redcap']`

Reply:
Okay and do we expect the following among others:

> ['xnat', 'upenn', 'redcap']

?

Reply:
> Okay and do we expect the following among others:

['xnat', 'upenn', 'redcap']


Expecting whatever the user provides to `--sources` argument in the `sync.py`. If `upenn` is one of them, `upenn_redcap` will be TRUE, and FALSE otherwise.

Reply:
Understood. A note about `upenn` would probably be required here:
https://github.com/AMP-SCZ/lochness/blob/master/docs/source/data_sources.rst

Reply:
Yeah... and bunch of other updates should be on the documentation as well. I've been focusing to finish off the working version of lochness for Pronet and Prescient as quickly as possible since we are expecting the first subject within a few weeks. I'll have to bug you a lot again once I start working on the documentation part.